gradle.properties

```diff
@@ -4,8 +4,8 @@ org.gradle.jvmargs=-Xmx1G
 # Fabric Properties
 	# check these on https://fabricmc.net/use
 	minecraft_version=1.15.2
-	yarn_mappings=1.15.2
-	loader_version=0.8.2
+	yarn_mappings=1.15.2+build.17
```

Comment:
No. Even if we did use Minecraft as a dependency, this would be incorrect.
Yarn mappings always follow `{minecraft_version}+build+{build_number}` format. If we were keeping track of the build number, this would be redundant.